eng/pipelines/installer/steps/helix.yml

```diff
@@ -0,0 +1,18 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  targetRid: ''
+  creator: ''
+  helixQueues: ''
+
+steps:
+  - template: /eng/common/core-templates/steps/send-to-helix.yml
+    parameters:
+      DisplayNamePrefix: Send to Helix
+      HelixProjectPath: src/installer/tests/helixpublish.proj
+      HelixProjectArguments: /p:Configuration=${{ parameters.buildConfig }} /p:TargetArchitecture=${{ parameters.archType }} /p:TargetRuntimeIdentifier=${{ parameters.targetRid }} /p:TargetOS=${{ parameters.osGroup }}
```

Comment:
Just curious, if we remove `TargetRuntimeIdentifier` from here, would it default to OutputRID as before?

Reply:
Actually, I think the host tests always use `OutputRID`, so `TargetRuntimeIdentifier` is not needed/used (and this is just from using the arguments from libraries tests as reference)

Reply:
On this note, I think `targetRid` (which gets passed to builds as `p:TargetRid`) shouldn't be needed in `eng/pipelines/*` in principle as in the code, we compute the OutputRID; except for `NonexistentRID` (`banana.24-x64` dummy RID test leg which is failing for several months) and `CentOS9` (`centos.9-x64` which is a non-portable RID).

cc @jkoritzinsky